libdirections-offboard/src/main/java/com/mapbox/navigation/route/offboard/router/NavigationOffboardRoute.kt

```diff
@@ -57,15 +60,21 @@ constructor(
                     DirectionsCriteria.ANNOTATION_CONGESTION,
                     DirectionsCriteria.ANNOTATION_DISTANCE
                 )
+                .routeOptions(RouteOptionsNavigation.builder()
```

Comment:
What about making the required parameters in `RouteOptionsNavigation` so we don’t need to add default `origin`, `destination` and `accessToken`? Right now, these are nullable in `RouteOptionsNavigation` e.g. https://github.com/mapbox/mapbox-navigation-android/blob/1bf6dee6d0ee6a413e7339b13d86c3f50b4d31a4/libnavigation-base/src/main/java/com/mapbox/navigation/base/route/model/RouteOptionsNavigation.kt#L24